### ISSUE TITLE
RFE: In MM Firing Phase Window please add option to show the TH Modifiers

### ISSUE THREAD

**Issue by scJazz2** [MegaMek/megamek#619]
0.43.3

Example from Firing Phase in a game...
Weapons fire for Cauldron-Born (Ebon Jaguar) A (Clan Smoke Jaguar)
    ER Large Laser at Thunderbolt TDR-4RR (The Corp); needs 11, rolls 9 : misses

Please add an option to show the TH mods. Perhaps something like this...
Weapons fire for Cauldron-Born (Ebon Jaguar) A (Clan Smoke Jaguar)
    ER Large Laser at Thunderbolt TDR-4RR (The Corp); needs 11, rolls 9 : misses
        To Hit = (Gunnery) +1, (Target Moved 5) +2, (Target Jumped) +1, (Attacker Jumped) +3, (Long Range) +4 = 11

**Comment by arlith** [comment]
I had an idea a while back about adding that information via a tooltip.  I'm not 100% certain how this would work out, but I was wondering if I could abuse HTML like I did to create a link to open a unit in the UnitDisplay.  I thought there was an RFE for this already somewhere though.

**Comment by scJazz2** [comment]
You mean like hover over the Roll and it pops up or Click on the Roll and it displays? Either would work... although I like showing it in the logs just for the sake of having it saved and visible after the game.

**Comment by arlith** [comment]
Yes, that's what I meant.  I feel like having the full display in the log will really bloat the log.  Ideally, I'd like it to be a hover-over tooltip, but I'm not sure that can happen.  A clickable hyper-link is possible, exactly how we have the links for units involved in the reports now.

**Comment by scJazz2** [comment]
@arlith I am not sure that bloating the log would be a bad thing actually. It would also provide significantly more data to be harvested for any log parsing applications that may occur in the future.

**Comment by scJazz2** [comment]
Actually, thinking about it further... I'd suggest a ShowDice() option. Showing the dice rolls for everything.
